cmd/launcher/launcher.go

```diff
@@ -148,8 +148,9 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 
 	// Need to set up the log shipper so that we can get the logger early
 	// and pass it to the various systems.
+	isShippingLogs := k.ObservabilityIngestServerURL() != ""
```

Comment:
Probably okay to start with it. Interesting question is whether it's possible to enable/disable/change the URL from the control server